pkgs/development/python-modules/material-color-utilities/default.nix

```diff
@@ -0,0 +1,25 @@
+{ stdenv, lib, buildPythonPackage, fetchPypi, pillow, regex, ... }:
+
+buildPythonPackage rec {
+  pname = "material-color-utilities-python";
+  version = "0.1.5";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-PG8C585wWViFRHve83z3b9NijHyV+iGY2BdMJpyVH64=";
+  };
+
+  propagatedBuildInputs = [
+    pillow
+    regex
+  ];
+
+  # No tests implemented.
+  doCheck = false;
+
+  meta = with lib; {
+    description = "Python port of material-color-utilities used for Material You colors";
```

Comment:
Missing homepage

Reply:
Done. Though, I chose the Pypi page instead of the [Git repo](https://github.com/avanishsubbiah/material-color-utilities-python) since the Git repo seems to be out of date and not properly tagged. Not exactly relevant for a homepage but might as well make it easier to see the preferred page whenever an update comes.